resources/css/index.css

```diff
@@ -0,0 +1,186 @@
+@font-face {
+  font-family: "Yeseva One";
+  src: url(../fonts/YesevaOne.otf);
+}
+
+@font-face {
+  font-family: "Lato";
+  src: url(../fonts/Lato.ttf);
+}
+
+@font-face {
+  font-family: "GenB";
+  src: url(../fonts/GenB.ttf);
+}
+
+body, html {
+  margin: 0;
+  padding: 0;
+}
+
+.wrapper {
+  margin: 0 auto;
+  width: 1600px;
+}
+
+.button {
+  box-sizing: border-box;
+  color: #fff;
+  display: inline-block;
+  font-size: 18px;
+  font-family: "Lato", sans-serif;
+  text-transform: uppercase;
+  text-align: center;
+  text-decoration: none;
+  padding: 15px 35px;
+  margin-top: 30px;
```

Comment:
I'd put buttons into `<div class="button-group"></div>` and add margin to this container. Or add `margin-bottom` to `.divider`.

Reply:
Fixed.